### ISSUE TITLE
Bugs in Colorado subtractions added in PR #2823

### ISSUE THREAD

**Issue by martinholmer** [PolicyEngine/policyengine-us#2909]
@PavelMakarchuk, When I execute an integration test for the Colorado income tax, I get this error:
```
File "//Users/mrh/anaconda3/envs/policyengine-us/lib/python3.10/site-packages/policyengine_core/simulations/simulation.py", line 450, in _calculate
    raise ValueError(f"Variable {variable_name} does not exist.")
    raise ValueError(f"Variable {variable_name} does not exist.")
ValueError: Variable charitable_contribution does not exist.
ValueError: Variable charitable_contribution does not exist.
```

I have no idea where **all** the problems are except that there are clearly misspelled/misnamed variable names in the `parameters/gov/states/co/tax/income/subtractions/subtractions.yaml` file.

Seems like you should be doing more testing to catch these kinds of problems.  Unit tests, by themselves, will never catch problems in the parameter files that contain lists of variables that are added.

**Comment by PavelMakarchuk** [comment]
this should be fixed by #2825

**Comment by PavelMakarchuk** [comment]
> this should be fixed by #2825 @martinholmer

I think this should have been fixed
